miner/worker.go

```diff
@@ -1275,11 +1357,18 @@ func (w *worker) postSideBlock(event core.ChainSideEvent) {
 	}
 }
 
+func ethIntToFloat(eth *big.Int) *big.Float {
+	if eth == nil {
+		return big.NewFloat(0)
+	}
+	return new(big.Float).Quo(new(big.Float).SetInt(eth), new(big.Float).SetInt(big.NewInt(params.Ether)))
```

Comment:
why the quo? is this for human readableness?

Reply:
yes

Reply:
worth noting: a bunch of these logs are temporary, and we'll delete before final merge